server/src/workspace/workspace-schema-builder/graphql-types/scalars/big-float.scalar.ts

```diff
@@ -5,17 +5,16 @@ export const BigFloatScalarType = new GraphQLScalarType({
   name: 'BigFloat',
   description:
     'A custom scalar type for representing big floating point numbers',
-  serialize(value: number): string {
-    return String(value);
-  },
-  parseValue(value: string): number {
+  serialize(value: string): number {
```

Comment:
We should reverse our interpretation of scalars as, we're passing arguments directly to `pg_graphql`